rkt/rkt.go

```diff
@@ -44,6 +44,7 @@ var (
 		Debug              bool
 		Help               bool
 		InsecureSkipVerify bool
+		TrustKeysFromHttps bool
```

Comment:
It can be used by all the fetch commands:
- `rkt fetch`
- `rkt run`
- `rkt prepare`

It is the same as `InsecureSkipVerify`, so if `TrustKeysFromHttps` becomes a local parameter, `InsecureSkipVerify` should be moved as well. If it is moved, the functions in `rkt/images.go` would need to get more parameters because it would not be a member of `globalFlags` anymore. Or we need a way to have common parameters between fetch commands.